platform/deploy.provision.inc

```diff
@@ -169,8 +169,24 @@ function drush_provision_drupal_provision_deploy() {
  * Implements drush_hook_post_COMMAND().
  */
 function drush_provision_drupal_post_provision_deploy() {
+  // Run registry-rebuild without cache-clear, if available.
+  if (function_exists('drush_registry_rebuild')) {
+    if (drush_drupal_major_version() < 8) {
```

Comment:
Why not `if (function_exists('drush_registry_rebuild') && drush_drupal_major_version() < 8) {`
?